tests/tests.py

```diff
@@ -0,0 +1,92 @@
+try:
+    from unittest.mock import create_autospec, Mock, patch, MagicMock
+except ImportError:
+    from mock import create_autospec, Mock, patch, MagicMock
+
+from reportportal_client.service import _convert_string, _list_to_payload, \
+    uri_join, _get_id, _get_msg, _get_data, _get_json, _get_messages, ReportPortalService
+
+import unittest
+from datetime import datetime
+
+
+class Test(unittest.TestCase):
+
+    def test_check_convert_to_string(self):
```

Comment:
Let's start adding docstrings everywhere. See PEP 257.

Reply:
tests usuallly are not describe, what you want to see in docstrings?